.gitignore

```diff
@@ -6,4 +6,8 @@ rebusurance.zip
 config.js
 .parcel-cache
 dist/
-local.config.js
+dist-shieldjs/
+download/
+src/local.config.js
+shieldlib/dist
+shieldlib/node_modules
```

Comment:
These are redundant.

Reply:
Why are these redundant?

Reply:
Unless you prefix a gitignore line with `/`, it matches at any depth. (Trailing `/` means "match a directory, not a file.")

| New line | Already covered by |
|----------|--------------------|
| `download/` | `download/` |
| `src/local.config.js` | `local.config.js` |
| `shieldlib/dist` | `dist/` |
| `shieldlib/node_modules` | `node_modules` |

I don't see `dist-shieldjs` as a redundancy, though, if it's still an ignore that you need.

Reply:
Thanks for the explanation!

Reply:
Resolved in ef3ca9a